tests/lib/image.sh

```diff
@@ -15,7 +15,7 @@ build_ubuntu_image() {
             # use go get so that ubuntu-image is built with current snapd
             # sources
             go get github.com/canonical/ubuntu-image/cmd/ubuntu-image
-            go install -tags 'withtestkeys' github.com/canonical/ubuntu-image/cmd/ubuntu-image
+            go install -tags 'withtestkeys' github.com/canonical/ubuntu-image/cmd/ubuntu-image@9196729ef8f14f530bec2c7512adea07e8f74da5
```

Comment:
Can we poke the upstreams of ubuntu-image to make more frequent and controlled releases? I see there are some tags on https://github.com/canonical/ubuntu-image/tags but I'm not sure if any of them is good enough for us.